### ISSUE TITLE
Cannot build docker for 3.4.4 anymore

### ISSUE THREAD

**Issue by danieljakots** [mastodon/mastodon#17384]
### Steps to reproduce the problem

docker build --pull --no-cache . -t mastodon:v3.4.4r1

### Expected behaviour

build finishes

### Actual behaviour

Step 27/30 : RUN cd ~ && 	OTP_SECRET=precompile_placeholder SECRET_KEY_BASE=precompile_placeholder rails assets:precompile && 	yarn cache clean  ---> Running in 36c498d5ff1d /opt/ruby/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/definition.rb:481:in `materialize': Could not find pkg-config-1.4.6, puma-5.3.2, rails-6.1.3.2, sprockets-3.7.2, thor-1.1.0, rack-2.2.3, hamlit-rails-0.2.3, pg-1.2.3, makara-0.5.0, pghero-2.8.1, dotenv-rails-2.7.6, aws-sdk-s3-1.95.1, fog-core-2.1.0, fog-openstack-0.3.10, paperclip-6.0.0, blurhash-0.1.5, active_model_serializers-0.10.12, addressable-2.7.0, bootsnap-1.6.0, browser-4.2.0, charlock_holmes-0.7.7, iso-639-0.3.5, chewy-5.2.0, cld3-3.4.2, devise-4.8.0, devise-two-factor-4.0.0, net-ldap-0.17.0, omniauth-cas-2.0.0, omniauth-saml-1.10.3, omniauth-1.9.1, omniauth-rails_csrf_protection-0.1.2, color_diff-0.1, discard-1.2.0, doorkeeper-5.5.1, ed25519-1.2.4, fast_blank-1.0.0, fastimage-2.2.3, hiredis-0.6.3, redis-namespace-1.8.1, htmlentities-4.3.4, http-4.4.1, http_accept_language-2.1.1, httplog-1.5.0, idn-ruby-0.1.0, kaminari-1.2.1, link_header-0.0.8, mime-types-3.3.1, nokogiri-1.11.6, nsa-0.2.8, oj-3.11.5, ox-2.14.4, parslet-2.0.0, parallel-1.20.1, posix-spawn-0.3.15, pundit-2.1.0, premailer-rails-1.11.1, rack-attack-6.5.0, rack-cors-1.1.1, rails-i18n-6.0.0, rails-settings-cached-0.6.6, redis-4.2.5, mario-redis-lock-1.2.1, rqrcode-2.0.0, ruby-progressbar-1.11.0, sanitize-5.2.3, scenic-1.5.4, sidekiq-6.2.1, sidekiq-scheduler-3.0.1, sidekiq-unique-jobs-7.0.11, sidekiq-bulk-0.2.0, simple-navigation-4.3.0, simple_form-5.1.0, sprockets-rails-3.2.2, stoplight-2.2.1, strong_migrations-0.7.6, tty-prompt-0.23.1, twitter-text-3.1.0, tzinfo-data-1.2021.1, webpacker-5.4.0, webpush-0.3.8, webauthn-3.0.0.alpha1, json-ld-3.1.9, json-ld-preloaded-3.1.5, rdf-normalize-0.4.0, private_address_check-0.5.0, lograge-0.11.2, concurrent-ruby-1.1.8, connection_pool-2.2.5, xorcist-1.1.2, resolv-0.1.0, nio4r-2.5.7, actioncable-6.1.3.2, actionmailbox-6.1.3.2, actionmailer-6.1.3.2, actionpack-6.1.3.2, actiontext-6.1.3.2, actionview-6.1.3.2, activejob-6.1.3.2, activemodel-6.1.3.2, activerecord-6.1.3.2, activestorage-6.1.3.2, activesupport-6.1.3.2, railties-6.1.3.2, hamlit-2.13.0, dotenv-2.7.6, aws-sdk-core-3.114.0, aws-sdk-kms-1.43.0, aws-sigv4-1.2.3, builder-3.2.4, excon-0.76.0, formatador-0.2.5, fog-json-1.2.0, ipaddress-0.8.3, mimemagic-0.3.10, terrapin-0.6.0, ffi-1.15.0, case_transform-0.2, jsonapi-renderer-0.2.2, public_suffix-4.0.6, msgpack-1.4.2, elasticsearch-7.10.1, elasticsearch-dsl-0.1.9, bcrypt-3.1.16, orm_adapter-0.5.0, responders-3.0.1, warden-1.2.9, attr_encrypted-3.1.0, rotp-6.2.0, ruby-saml-1.11.0, hashie-4.1.0, http-cookie-1.0.3, http-form_data-2.3.0, http-parser-1.2.1, rainbow-3.0.0, kaminari-actionview-1.2.1, kaminari-activerecord-1.2.1, kaminari-core-1.2.1, mime-types-data-3.2020.0512, mini_portile2-2.5.2, racc-1.5.2, statsd-ruby-1.5.0, premailer-1.14.2, i18n-1.8.10, chunky_png-1.4.0, rqrcode_core-1.0.0, crass-1.0.6, nokogumbo-2.0.4, e2mmap-0.1.0, rufus-scheduler-3.6.0, thwait-0.2.0, tilt-2.0.10, brpoplpush-redis_script-0.1.2, pastel-0.8.0, tty-reader-0.9.0, unf-0.1.4, tzinfo-2.0.4, rack-proxy-0.6.5, semantic_range-3.0.0, hkdf-0.3.0, jwt-2.2.2, android_key_attestation-0.3.0, awrence-1.1.1, bindata-2.4.8, cbor-0.5.9.6, cose-1.0.0, openssl-2.2.0, safety_net_attestation-0.4.0, securecompare-1.0.0, tpm-key_attestation-0.9.0, json-canonicalization-0.2.1, multi_json-1.15.0, rdf-3.1.13, request_store-1.5.0, websocket-driver-0.7.3, mail-2.7.1, rails-dom-testing-2.0.3, rack-test-1.1.0, rails-html-sanitizer-1.3.0, erubi-1.10.0, globalid-0.4.2, marcel-1.0.1, mini_mime-1.0.3, minitest-5.14.4, zeitwerk-2.4.2, method_source-1.0.0, rake-13.0.3, temple-0.8.2, aws-eventstream-1.1.1, aws-partitions-1.465.0, jmespath-1.4.0, climate_control-0.2.0, elasticsearch-api-7.10.1, elasticsearch-transport-7.10.1, encryptor-3.0.0, domain_name-0.5.20190701, ffi-compiler-1.0.1, net-ftp-0.1.2, css_parser-1.7.1, fugit-1.3.9, tty-color-0.6.0, tty-cursor-0.7.1, tty-screen-0.8.1, wisper-2.0.1, unf_ext-0.0.7.7, openssl-signature_algorithm-0.4.0, hamster-3.0.0, websocket-extensions-0.1.5, loofah-2.9.1, faraday-1.3.0, net-protocol-0.1.0, time-0.1.0, et-orbi-1.2.4, raabro-1.3.3, faraday-net_http-1.0.1, multipart-post-2.1.1, ruby2_keywords-0.0.4 in any of the sources (Bundler::GemNotFound) 	from /opt/ruby/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/definition.rb:190:in `specs' 	from /opt/ruby/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/definition.rb:238:in `specs_for' 	from /opt/ruby/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/runtime.rb:18:in `setup' 	from /opt/ruby/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler.rb:151:in `setup' 	from /opt/ruby/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/setup.rb:20:in `block in <top (required)>' 	from /opt/ruby/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/ui/shell.rb:136:in `with_level' 	from /opt/ruby/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/ui/shell.rb:88:in `silence' 	from /opt/ruby/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/setup.rb:20:in `<top (required)>' 	from /opt/ruby/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require' 	from /opt/ruby/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require' 	from /opt/mastodon/config/boot.rb:8:in `<top (required)>' 	from /opt/mastodon/bin/rails:3:in `require_relative' 	from /opt/mastodon/bin/rails:3:in `<main>' The command '/bin/bash -o pipefail -c cd ~ && 	OTP_SECRET=precompile_placeholder SECRET_KEY_BASE=precompile_placeholder rails assets:precompile && 	yarn cache clean' returned a non-zero code: 1

(I can't find how to make this output more readable :()

### Specifications

$ git status
HEAD detached at v3.4.4
nothing to commit, working tree clean

I reproduce the problem on two different computers

**Comment by danieljakots** [comment]
I built HEAD is on main, 166cc5b and it finished fine.

Here's what it looks like:
Step 29/32 : RUN cd ~ && 	OTP_SECRET=precompile_placeholder SECRET_KEY_BASE=precompile_placeholder rails assets:precompile && 	yarn cache clean
 ---> Running in 3469c6962cfa
yarn install v1.22.17
[1/6] Validating package.json...
[2/6] Resolving packages...
[3/6] Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
^[1info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
warning redis@4.0.2: The engine "typescript" appears to be invalid.
[4/6] Linking dependencies...
warning " > react-redux-loading-bar@4.0.8" has incorrect peer dependency "react-redux@^3.0.0 || ^4.0.0 || ^5.0.0".
warning Workspaces can only be enabled in private projects.
[5/6] Building fresh packages...
[6/6] Cleaning modules...
Done in 101.31s.
I, [2022-01-27T02:59:10.254220 #7]  INFO -- : Writing /opt/mastodon/public/assets/doorkeeper/admin/application-a644908e7bab54fb749be0f59fb64a7480bbf9c4c2b79d4a65791cb7ab4d8730.css
I, [2022-01-27T02:59:10.255164 #7]  INFO -- : Writing /opt/mastodon/public/assets/doorkeeper/admin/application-a644908e7bab54fb749be0f59fb64a7480bbf9c4c2b79d4a65791cb7ab4d8730.css.gz
I, [2022-01-27T02:59:10.267008 #7]  INFO -- : Writing /opt/mastodon/public/assets/doorkeeper/application-c93dac2ad9d65e3393e0e2c958481e86ef7a5e5b0f6ce406842a7b99b25a4850.css
I, [2022-01-27T02:59:10.267119 #7]  INFO -- : Writing /opt/mastodon/public/assets/doorkeeper/application-c93dac2ad9d65e3393e0e2c958481e86ef7a5e5b0f6ce406842a7b99b25a4850.css.gz
I, [2022-01-27T02:59:10.269232 #7]  INFO -- : Writing /opt/mastodon/public/assets/pghero/favicon-db10337a56c45eb43c22ff5019546b520fa22c7281d4d385f235cbca67ed26bb.png
I, [2022-01-27T02:59:10.580783 #7]  INFO -- : Writing /opt/mastodon/public/assets/pghero/application-a60bf0a452ed064fef3594cf52a4c998712da7c76150f890f4eaa644f59671e4.js
I, [2022-01-27T02:59:10.580914 #7]  INFO -- : Writing /opt/mastodon/public/assets/pghero/application-a60bf0a452ed064fef3594cf52a4c998712da7c76150f890f4eaa644f59671e4.js.gz
I, [2022-01-27T02:59:10.589232 #7]  INFO -- : Writing /opt/mastodon/public/assets/pghero/application-c31338f656687c1d733bb0f48d40acd076e24060f3dcff83b34870e4ccc2789d.css
I, [2022-01-27T02:59:10.589347 #7]  INFO -- : Writing /opt/mastodon/public/assets/pghero/application-c31338f656687c1d733bb0f48d40acd076e24060f3dcff83b34870e4ccc2789d.css.gz
Compiling...
Compiled all packs in /opt/mastodon/public/packs
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating

yarn cache v1.22.17
success Cleared cache.
Done in 1.58s.
Removing intermediate container 3469c6962cfa
 ---> d2f8c957b52b

**Comment by ClearlyClaire** [comment]
I'm not sure what's going on, but it seems like it may be missing a `bundle install`. This should have been done in an earlier step of the `Dockerfile`, though:
https://github.com/mastodon/mastodon/blob/fd868f8ca0b13afdb3f6fb9a3be063ccec7e1349/Dockerfile#L56-L60

**Comment by danieljakots** [comment]
This step seems to have gone fine:

```
Step 10/30 : RUN cd /opt/mastodon &&   bundle config set deployment 'true' &&   bundle config set without 'development test' && 	bundle install -j"$(nproc)" && 	yarn install --pure-lockfile
 ---> Running in 69aad96cd314
Don't run Bundler as root. Bundler can ask for sudo if it is needed, and
installing your bundle as root will break this application for all non-root
users on this machine.
Fetching gem metadata from https://rubygems.org/.........
Fetching rake 13.0.3
Installing rake 13.0.3
Fetching concurrent-ruby 1.1.8
Fetching minitest 5.14.4
Fetching builder 3.2.4
Fetching erubi 1.10.0
Fetching net-protocol 0.1.0
Fetching zeitwerk 2.4.2
Fetching racc 1.5.2
Fetching time 0.1.0
Installing net-protocol 0.1.0
Installing time 0.1.0
Installing zeitwerk 2.4.2
Installing erubi 1.10.0
Fetching crass 1.0.6
Fetching rack 2.2.3
Fetching nio4r 2.5.7
Fetching websocket-extensions 0.1.5
Installing builder 3.2.4
Installing minitest 5.14.4
Fetching marcel 1.0.1
Fetching mini_mime 1.0.3
Installing crass 1.0.6
Installing concurrent-ruby 1.1.8
Fetching jsonapi-renderer 0.2.2
Installing mini_mime 1.0.3
Fetching public_suffix 4.0.6
Installing websocket-extensions 0.1.5
Installing nio4r 2.5.7 with native extensions
Installing rack 2.2.3
Installing jsonapi-renderer 0.2.2
Fetching android_key_attestation 0.3.0
Fetching encryptor 3.0.0
Installing marcel 1.0.1
Installing android_key_attestation 0.3.0
Installing racc 1.5.2 with native extensions
Installing public_suffix 4.0.6
Installing encryptor 3.0.0
Fetching awrence 1.1.1
Fetching aws-eventstream 1.1.1
Installing awrence 1.1.1
Installing aws-eventstream 1.1.1
Fetching aws-partitions 1.465.0
Fetching jmespath 1.4.0
Fetching bcrypt 3.1.16
Installing aws-partitions 1.465.0
Fetching bindata 2.4.8
Installing jmespath 1.4.0
Installing bcrypt 3.1.16 with native extensions
Installing bindata 2.4.8
Fetching ffi 1.15.0
Fetching msgpack 1.4.2
Fetching browser 4.2.0
Fetching redis 4.2.5
Using bundler 2.3.6
Fetching cbor 0.5.9.6
Installing browser 4.2.0
Installing msgpack 1.4.2 with native extensions
Installing redis 4.2.5
Fetching charlock_holmes 0.7.7
Installing ffi 1.15.0 with native extensions
Installing charlock_holmes 0.7.7 with native extensions
Fetching multi_json 1.15.0
Installing cbor 0.5.9.6 with native extensions
Installing multi_json 1.15.0
Fetching faraday-net_http 1.0.1
Installing faraday-net_http 1.0.1
Fetching multipart-post 2.1.1
Installing multipart-post 2.1.1
Fetching ruby2_keywords 0.0.4
Installing ruby2_keywords 0.0.4
Fetching elasticsearch-dsl 0.1.9
Installing elasticsearch-dsl 0.1.9
Fetching chunky_png 1.4.0
Installing chunky_png 1.4.0
Fetching climate_control 0.2.0
Installing climate_control 0.2.0
Fetching color_diff 0.1
Installing color_diff 0.1
Fetching connection_pool 2.2.5
Installing connection_pool 2.2.5
Fetching openssl-signature_algorithm 0.4.0
Installing openssl-signature_algorithm 0.4.0
Fetching orm_adapter 0.5.0
Installing orm_adapter 0.5.0
Fetching method_source 1.0.0
Installing method_source 1.0.0
Fetching thor 1.1.0
Installing thor 1.1.0
Fetching rotp 6.2.0
Installing rotp 6.2.0
Fetching unf_ext 0.0.7.7
Fetching dotenv 2.7.6
Installing unf_ext 0.0.7.7 with native extensions
Installing dotenv 2.7.6
Fetching e2mmap 0.1.0
Installing e2mmap 0.1.0
Fetching ed25519 1.2.4
Installing ed25519 1.2.4 with native extensions
Fetching excon 0.76.0
Installing excon 0.76.0
Fetching fast_blank 1.0.0
Installing fast_blank 1.0.0 with native extensions
Fetching fastimage 2.2.3
Installing fastimage 2.2.3
Fetching formatador 0.2.5
Installing formatador 0.2.5
Fetching mime-types-data 3.2020.0512
Installing mime-types-data 3.2020.0512
Fetching ipaddress 0.8.3
Installing ipaddress 0.8.3
Fetching raabro 1.3.3
Installing raabro 1.3.3
Fetching temple 0.8.2
Installing temple 0.8.2
Fetching tilt 2.0.10
Installing tilt 2.0.10
Fetching hashie 4.1.0
Installing hashie 4.1.0
Fetching hiredis 0.6.3
Installing hiredis 0.6.3 with native extensions
Fetching hkdf 0.3.0
Installing hkdf 0.3.0
Fetching htmlentities 4.3.4
Installing htmlentities 4.3.4
Fetching http-form_data 2.3.0
Installing http-form_data 2.3.0
Fetching http_accept_language 2.1.1
Installing http_accept_language 2.1.1
Fetching rainbow 3.0.0
Installing rainbow 3.0.0
Fetching idn-ruby 0.1.0
Installing idn-ruby 0.1.0 with native extensions
Fetching iso-639 0.3.5
Installing iso-639 0.3.5
Fetching json-canonicalization 0.2.1
Installing json-canonicalization 0.2.1
Fetching link_header 0.0.8
Installing link_header 0.0.8
Fetching jwt 2.2.2
Installing jwt 2.2.2
Fetching kaminari-core 1.2.1
Installing kaminari-core 1.2.1
Fetching net-ldap 0.17.0
Installing net-ldap 0.17.0
Fetching statsd-ruby 1.5.0
Installing statsd-ruby 1.5.0
Fetching oj 3.11.5
Installing oj 3.11.5 with native extensions
Fetching openssl 2.2.0
Installing openssl 2.2.0 with native extensions
Fetching ox 2.14.4
Installing ox 2.14.4 with native extensions
Fetching parallel 1.20.1
Installing parallel 1.20.1
Fetching parslet 2.0.0
Installing parslet 2.0.0
Fetching tty-color 0.6.0
Installing tty-color 0.6.0
Fetching pg 1.2.3
Installing pg 1.2.3 with native extensions
Fetching pkg-config 1.4.6
Installing pkg-config 1.4.6
Fetching posix-spawn 0.3.15
Installing posix-spawn 0.3.15 with native extensions
Fetching private_address_check 0.5.0
Installing private_address_check 0.5.0
Fetching resolv 0.1.0
Installing resolv 0.1.0
Fetching rqrcode_core 1.0.0
Installing rqrcode_core 1.0.0
Fetching ruby-progressbar 1.11.0
Installing ruby-progressbar 1.11.0
Fetching securecompare 1.0.0
Installing securecompare 1.0.0
Fetching semantic_range 3.0.0
Installing semantic_range 3.0.0
Fetching stoplight 2.2.1
Installing stoplight 2.2.1
Fetching tty-cursor 0.7.1
Installing tty-cursor 0.7.1
Fetching tty-screen 0.8.1
Installing tty-screen 0.8.1
Fetching wisper 2.0.1
Installing wisper 2.0.1
Fetching xorcist 1.1.2
Installing xorcist 1.1.2 with native extensions
Fetching net-ftp 0.1.2
Installing net-ftp 0.1.2
Fetching mail 2.7.1
Installing mail 2.7.1
Fetching websocket-driver 0.7.3
Installing websocket-driver 0.7.3 with native extensions
Fetching i18n 1.8.10
Installing i18n 1.8.10
Fetching tzinfo 2.0.4
Installing tzinfo 2.0.4
Fetching hamster 3.0.0
Installing hamster 3.0.0
Fetching attr_encrypted 3.1.0
Installing attr_encrypted 3.1.0
Fetching aws-sigv4 1.2.3
Installing aws-sigv4 1.2.3
Fetching addressable 2.7.0
Installing addressable 2.7.0
Fetching rack-test 1.1.0
Installing rack-test 1.1.0
Fetching warden 1.2.9
Installing warden 1.2.9
Fetching request_store 1.5.0
Installing request_store 1.5.0
Fetching rack-attack 6.5.0
Installing rack-attack 6.5.0
Fetching rack-cors 1.1.1
Installing rack-cors 1.1.1
Fetching rack-proxy 0.6.5
Installing rack-proxy 0.6.5
Fetching sprockets 3.7.2
Installing sprockets 3.7.2
Fetching brpoplpush-redis_script 0.1.2
Installing brpoplpush-redis_script 0.1.2
Fetching mario-redis-lock 1.2.1
Installing mario-redis-lock 1.2.1
Fetching redis-namespace 1.8.1
Installing redis-namespace 1.8.1
Fetching elasticsearch-api 7.10.1
Installing elasticsearch-api 7.10.1
Fetching faraday 1.3.0
Installing faraday 1.3.0
Fetching terrapin 0.6.0
Installing terrapin 0.6.0
Fetching sidekiq 6.2.1
Installing sidekiq 6.2.1
Fetching tpm-key_attestation 0.9.0
Installing tpm-key_attestation 0.9.0
Fetching puma 5.3.2
Installing puma 5.3.2 with native extensions
Fetching thwait 0.2.0
Installing thwait 0.2.0
Fetching mime-types 3.3.1
Installing mime-types 3.3.1
Fetching hamlit 2.13.0
Installing hamlit 2.13.0 with native extensions
Fetching omniauth 1.9.1
Installing omniauth 1.9.1
Fetching httplog 1.5.0
Installing httplog 1.5.0
Fetching bootsnap 1.6.0
Installing bootsnap 1.6.0 with native extensions
Fetching safety_net_attestation 0.4.0
Installing safety_net_attestation 0.4.0
Fetching webpush 0.3.8
Installing webpush 0.3.8
Fetching cose 1.0.0
Installing cose 1.0.0
Fetching blurhash 0.1.5
Installing blurhash 0.1.5 with native extensions
Fetching cld3 3.4.2
Fetching ffi-compiler 1.0.1
Installing ffi-compiler 1.0.1
Fetching unf 0.1.4
Fetching pastel 0.8.0
Installing pastel 0.8.0
Fetching rqrcode 2.0.0
Fetching tty-reader 0.9.0
Installing unf 0.1.4
Fetching mini_portile2 2.5.2
Installing tty-reader 0.9.0
Fetching activesupport 6.1.3.2
Installing rqrcode 2.0.0
Fetching et-orbi 1.2.4
Installing mini_portile2 2.5.2
Fetching tzinfo-data 1.2021.1
Installing et-orbi 1.2.4
Fetching rdf 3.1.13
Installing activesupport 6.1.3.2
Fetching aws-sdk-core 3.114.0
Installing rdf 3.1.13
Fetching css_parser 1.7.1
Installing aws-sdk-core 3.114.0
Fetching elasticsearch-transport 7.10.1
Installing tzinfo-data 1.2021.1
Installing css_parser 1.7.1
Installing elasticsearch-transport 7.10.1
Fetching sidekiq-bulk 0.2.0
Fetching sidekiq-unique-jobs 7.0.11
Installing sidekiq-bulk 0.2.0
Fetching fog-core 2.1.0
Installing sidekiq-unique-jobs 7.0.11
Installing fog-core 2.1.0
Fetching webauthn 3.0.0.alpha1
Fetching http-parser 1.2.1
Fetching domain_name 0.5.20190701
Installing webauthn 3.0.0.alpha1
Fetching twitter-text 3.1.0
Installing http-parser 1.2.1 with native extensions
Installing domain_name 0.5.20190701
Fetching tty-prompt 0.23.1
Installing twitter-text 3.1.0
Fetching nokogiri 1.11.6 (x86_64-linux)
Installing tty-prompt 0.23.1
Fetching fugit 1.3.9
Installing fugit 1.3.9
Fetching globalid 0.4.2
Fetching activemodel 6.1.3.2
Installing globalid 0.4.2
Installing activemodel 6.1.3.2
Fetching case_transform 0.2
Fetching nsa 0.2.8
Installing case_transform 0.2
Fetching pundit 2.1.0
Installing nsa 0.2.8
Fetching simple-navigation 4.3.0
Fetching json-ld 3.1.9
Installing pundit 2.1.0
Fetching rdf-normalize 0.4.0
Installing simple-navigation 4.3.0
Fetching aws-sdk-kms 1.43.0
Installing rdf-normalize 0.4.0
Fetching premailer 1.14.2
Fetching elasticsearch 7.10.1
Installing aws-sdk-kms 1.43.0
Fetching fog-json 1.2.0
Installing premailer 1.14.2
Installing elasticsearch 7.10.1
Fetching http-cookie 1.0.3
Fetching rufus-scheduler 3.6.0
Installing fog-json 1.2.0
Fetching activejob 6.1.3.2
Fetching activerecord 6.1.3.2
Installing http-cookie 1.0.3
Installing rufus-scheduler 3.6.0
Fetching aws-sdk-s3 1.95.1
Fetching chewy 5.2.0
Installing json-ld 3.1.9
Installing activejob 6.1.3.2
Fetching fog-openstack 0.3.10
Fetching sidekiq-scheduler 3.0.1
Installing sidekiq-scheduler 3.0.1
Installing activerecord 6.1.3.2
Fetching json-ld-preloaded 3.1.5
Installing chewy 5.2.0
Fetching http 4.4.1
Installing fog-openstack 0.3.10
Installing http 4.4.1
Installing aws-sdk-s3 1.95.1
Installing json-ld-preloaded 3.1.5
Fetching discard 1.2.0
Fetching makara 0.5.0
Fetching kaminari-activerecord 1.2.1
Installing discard 1.2.0
Installing makara 0.5.0
Fetching pghero 2.8.1
Installing kaminari-activerecord 1.2.1
Fetching strong_migrations 0.7.6
Installing strong_migrations 0.7.6
Installing pghero 2.8.1
Installing nokogiri 1.11.6 (x86_64-linux)
Installing cld3 3.4.2 with native extensions
Fetching rails-dom-testing 2.0.3
Fetching mimemagic 0.3.10
Fetching nokogumbo 2.0.4
Fetching ruby-saml 1.11.0
Fetching loofah 2.9.1
Fetching omniauth-cas 2.0.0
Installing rails-dom-testing 2.0.3
Installing mimemagic 0.3.10 with native extensions
Installing omniauth-cas 2.0.0
Installing loofah 2.9.1
Fetching rails-html-sanitizer 1.3.0
Installing nokogumbo 2.0.4 with native extensions
Installing rails-html-sanitizer 1.3.0
Fetching paperclip 6.0.0
Installing ruby-saml 1.11.0
Fetching actionview 6.1.3.2
Fetching omniauth-saml 1.10.3
Installing actionview 6.1.3.2
Installing paperclip 6.0.0
Installing omniauth-saml 1.10.3
Fetching actionpack 6.1.3.2
Fetching kaminari-actionview 1.2.1
Installing kaminari-actionview 1.2.1
Fetching kaminari 1.2.1
Installing actionpack 6.1.3.2
Installing kaminari 1.2.1
Fetching actioncable 6.1.3.2
Fetching activestorage 6.1.3.2
Fetching actionmailer 6.1.3.2
Fetching omniauth-rails_csrf_protection 0.1.2
Fetching railties 6.1.3.2
Fetching active_model_serializers 0.10.12
Installing omniauth-rails_csrf_protection 0.1.2
Fetching sprockets-rails 3.2.2
Installing actionmailer 6.1.3.2
Fetching simple_form 5.1.0
Installing activestorage 6.1.3.2
Fetching premailer-rails 1.11.1
Installing active_model_serializers 0.10.12
Installing actioncable 6.1.3.2
Installing sprockets-rails 3.2.2
Fetching actionmailbox 6.1.3.2
Installing simple_form 5.1.0
Fetching actiontext 6.1.3.2
Installing premailer-rails 1.11.1
Installing actionmailbox 6.1.3.2
Installing actiontext 6.1.3.2
Installing railties 6.1.3.2
Fetching responders 3.0.1
Fetching doorkeeper 5.5.1
Fetching dotenv-rails 2.7.6
Fetching hamlit-rails 0.2.3
Fetching rails 6.1.3.2
Fetching lograge 0.11.2
Installing rails 6.1.3.2
Fetching rails-i18n 6.0.0
Installing dotenv-rails 2.7.6
Installing hamlit-rails 0.2.3
Installing lograge 0.11.2
Installing responders 3.0.1
Fetching scenic 1.5.4
Installing doorkeeper 5.5.1
Fetching webpacker 5.4.0
Fetching rails-settings-cached 0.6.6
Fetching devise 4.8.0
Installing scenic 1.5.4
Installing rails-i18n 6.0.0
Installing rails-settings-cached 0.6.6
Installing devise 4.8.0
Installing webpacker 5.4.0
Fetching devise-two-factor 4.0.0
Installing devise-two-factor 4.0.0
Fetching sanitize 5.2.3
Installing sanitize 5.2.3
Bundle complete! 124 Gemfile dependencies, 217 gems now installed.
Gems in the groups 'development' and 'test' were not installed.
Bundled gems are installed into `./vendor/bundle`
Post-install message from encryptor:



Please be aware that Encryptor v2.0.0 had a major security bug when using AES-*-GCM algorithms.

By default You will not be able to decrypt data that was previously encrypted using an AES-*-GCM algorithm.

Please see the README and https://github.com/attr-encrypted/encryptor/pull/22 for more information.


Post-install message from attr_encrypted:



WARNING: Several insecure default options and features were deprecated in attr_encrypted v2.0.0.

Additionally, there was a bug in Encryptor v2.0.0 that insecurely encrypted data when using an AES-*-GCM algorithm.

This bug was fixed but introduced breaking changes between v2.x and v3.x.

Please see the README for more information regarding upgrading to attr_encrypted v3.0.0.


Post-install message from doorkeeper:
Starting from 5.5.0 RC1 Doorkeeper requires client authentication for Resource Owner Password Grant
as stated in the OAuth RFC. You have to create a new OAuth client (Doorkeeper::Application) if you didn't
have it before and use client credentials in HTTP Basic auth if you previously used this grant flow without
client authentication. 

To opt out of this you could set the "skip_client_authentication_for_password_grant" configuration option
to "true", but note that this is in violation of the OAuth spec and represents a security risk.

Read https://github.com/doorkeeper-gem/doorkeeper/issues/561#issuecomment-612857163 for more details.
Post-install message from paperclip:
##################################################
#  NOTE FOR UPGRADING FROM 4.3.0 OR EARLIER      #
##################################################

Paperclip is now compatible with aws-sdk >= 2.0.0.

If you are using S3 storage, aws-sdk >= 2.0.0 requires you to make a few small
changes:

* You must set the `s3_region`
* If you are explicitly setting permissions anywhere, such as in an initializer,
  note that the format of the permissions changed from using an underscore to
  using a hyphen. For example, `:public_read` needs to be changed to
  `public-read`.

For a walkthrough of upgrading from 4 to 5 and aws-sdk >= 2.0 you can watch
http://rubythursday.com/episodes/ruby-snack-27-upgrade-paperclip-and-aws-sdk-in-prep-for-rails-5
Post-install message from sidekiq-unique-jobs:
IMPORTANT!

Automatic configuration of the sidekiq middelware is no longer done.
Please see: https://github.com/mhenrixon/sidekiq-unique-jobs/blob/master/README.md#add-the-middleware

This version deprecated the following sidekiq_options

  - sidekiq_options lock_args: :method_name

It is now configured with:

  - sidekiq_options lock_args_method: :method_name

This is also true for `Sidekiq.default_worker_options`

We also deprecated the global configuration options:
  - default_lock_ttl
  - default_lock_ttl=
  - default_lock_timeout
  - default_lock_timeout=

The new methods to use are:
  - lock_ttl
  - lock_ttl=
  - lock_timeout
  - lock_timeout=
yarn install v1.22.17
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
[4/5] Linking dependencies...
warning " > react-redux-loading-bar@4.0.8" has incorrect peer dependency "react-redux@^3.0.0 || ^4.0.0 || ^5.0.0".
[5/5] Building fresh packages...
Done in 29.71s.
Removing intermediate container 69aad96cd314
 ---> 2fe64ff07bce
```

**Comment by ClearlyClaire** [comment]
This seems to be a duplicate of https://github.com/mastodon/mastodon/issues/17246
A solution had been found there, I will investigate it later to maybe backport the fix!

**Comment by danieljakots** [comment]
Hand-applying fad37dd1bc8504fe6f2939f5d935cc1e5d264458 indeed solved the problem. Thank you very much!

I don't know if you want to keep it open until you finish your investigation, but you can close this issue either way ;)

**Comment by ClearlyClaire** [comment]
Fixed in v3.4.5!
